.prettierrc

```diff
@@ -0,0 +1,14 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
```

Comment:
How do you feel about 4 (like demos/whisper-base/streaming_processor.js used to be)? Logical block structures really stand out more, and you can discern the opening and corresponding closing braces, making it easier to visually scan quickly. (and yes, I'm biased because 8 of the large codebases I worked on used 4, with Chromium and ORT being odd 2 column outliers).